lib/rubocop/cop/util.rb

```diff
@@ -148,6 +148,10 @@ def move_pos(src, pos, step, condition, regexp)
         pos
       end
 
+      def looking_at(range, regexp)
+        range.source_buffer.source.match(/\G#{regexp}/, range.begin_pos)
```

Comment:
`\G` is only useful when a regexp is passed to `scan` or a similar method. It won't do anything here.